app/views/settings/_page_header.html.erb

```diff
@@ -0,0 +1,26 @@
+<div id="settings-page-header-desktop" class="flex gap-8 border-b p-8 border-line-colour">
```

Comment:
rename id to 'settings-page-header' to the same as per the above comment.

Reply:
id is renamed